nsv13/code/modules/mob/mob.dm

```diff
@@ -4,3 +4,9 @@
 /mob/Move(atom/newloc, direct=0)
 	. = ..()
 	find_overmap()
+
+// There's so many edge cases for player overmap escapes that it's just cleaner to universally check mob forcemoves.
+// use doMove if you must get around this for whatever reason
+/mob/living/forceMove(atom/destination)
+	destination = GetSafeTurf()
```

Comment:
should this be destination.GetSafeTurf() ?
it looks like if someone was not on the overmap, then lines 27 and 28 would just teleport them to the turf they're currently on, right?

Reply:
*scream

I forgot I moved the order around. Cheers